src/Runner.Worker/ExecutionContext.cs

```diff
@@ -365,11 +388,9 @@ public void RegisterPostJobStep(IStep step)
                 child._logger.Setup(_mainTimelineId, recordId);
             }
 
-            child.IsEmbedded = isEmbedded;
             child.StepTelemetry.StepId = recordId;
             child.StepTelemetry.Stage = stage.ToString();
-            child.StepTelemetry.IsEmbedded = isEmbedded;
```

Comment:
`child.IsEmbedded` and `child.StepTelemetry.IsEmbedded` are now initialized in the workhorse constructor.